microprofile/reactive-streams-operators-smallrye/extension/src/main/java/org/wildfly/extension/microprofile/reactive/streams/operators/_private/MicroProfileReactiveStreamsOperatorsLogger.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2020, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.wildfly.extension.microprofile.reactive.streams.operators._private;
+
+import static org.jboss.logging.Logger.Level.INFO;
+
+import org.jboss.as.server.deployment.DeploymentUnitProcessingException;
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * Log messages for WildFly microprofile-health-smallrye Extension.
+ *
+ * @author <a href="kkhan@redhat.com">Kabir Khan</a> (c) 2019 Red Hat inc.
+ */
+@MessageLogger(projectCode = "WFLYRXSTOPS", length = 4)
+public interface MicroProfileReactiveStreamsOperatorsLogger extends BasicLogger {
+
+    MicroProfileReactiveStreamsOperatorsLogger LOGGER = Logger.getMessageLogger(MicroProfileReactiveStreamsOperatorsLogger.class, "org.wildfly.extension.microprofile.reactive-streams-operators-smallrye");
+
+    /**
+     * Logs an informational message indicating the subsystem is being activated.
+     */
+    @LogMessage(level = INFO)
+    @Message(id = 1, value = "Activating Eclipse MicroProfile Reactive Streams Operators Subsystem")
```

Comment:
Since you've added two of these kinds of messages, I hereby assign you ownership of getting #13503 reviewed, amended as needed, and merged.